services/src/main/groovy/org/jfrog/artifactory/client/model/xray/settings/impl/XraySettingsImpl.groovy

```diff
@@ -23,17 +22,17 @@ public void setXrayIndex(Boolean xrayIndex) {
 
     @Override
     public boolean equals(Object o) {
```

Comment:
A few semi-colons (;) were removed in this method.
Let's add them back.

Reply:
added back